Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG PY_VER
```

Comment:
I have `Dockerfile.test` and `Dockerfile.dev` in my workflow, not my element. Should this be in the element?

Reply:
Thanks @CBroz1.  The Docker and Compose files here are for the `element` release on pypi.